### PR TITLE
Cus 3005 mountain crest skip a pay fee toloan

### PR DESCRIPTION
**Pull request by rrobison44**
New loan payment skip feature for MountainCrest CU.  Feature allows member to apply skip payment fee (if being charged) to each of the individual loans being skipped vs. choosing a share for the fee source.  If the new CFG (FLA) is set to TRUE member will have the choice to either choose a share for the fee source or choose to apply the fee to the skipped loans.

[Jira CUS-3005](https://banno-jha.atlassian.net/browse/CUS-3005)

**Note: This PR will be closed.  It is being replaced by [PR 260](https://github.com/Banno/custom-powerons/pull/260) in the `custom-powerons` repo.  These changes will not be included in the baseline/std code in the `banno-powerons` repo.

### CODE REVIEW

**Review by jucarson — APPROVED** [review]
Looks good Robby!

**Review by tkainz — APPROVED** [review]
Everything looks fine Robby.  Thanks

File: loan-payment-skip/loan-payment-skip-json-contract.md [hunk 1]
```diff
@@ -9,94 +11,105 @@ To get things started, the client will send an initial request structured like s
...
+| Invalid loan id: [loan id]                                         | This set of errors is for skip payment processing errors                                                         |
+| Attempt to skip a payment on an ineligible loan: [loan id]         |                                                                                                                  |
+| Invalid fee share id: [share id]                                   |                                                                                                                  |
+| Loan Record Update Failed. Error: [file maint system error]        |                                                                                                                  |
+| Tracking Record Create Failed. Error: [file maint system error]    |                                                                                                                  |
+| Skip Pay Fee Post Failed. Error: [tran posting system error]       |                                                                                                                  |
```
Comment (jordan-keenan) [inline loan-payment-skip/loan-payment-skip-json-contract.md:115]: change this section to somehow show:
Share Skip Pay Fee Post Failed. Error: [tran posting system error]
and
Loan Skip Pay Fee Post Failed. Error: [tran posting system error]

**Review by jordan-keenan — APPROVED** [review]
@rrobison44 Code Changes look good! Thank you

**Comment by rrobison44** [comment]
Closing PR since a PR in `custom-powerons` is replacing this.

**Pull request closed without merging**